### DISCUSSION TITLE
AP szám megjelenítése

### DISCUSSION THREAD

**Discussion by wlln22** [nav-gov-hu/eRECEIPT#37] [Kérdések és válaszok]
Továbbra is elvárás, hogy jól látható helyen, jól olvashatóan meg kell jeleníteni az e-pénztárgép AP számát. Felhőalapú pénztárgép esetén, ha a hardver mobiltelefon, ez hogyan oldható meg?

**Comment by BatariZ** [comment]
Az időközben megjelent rendelet (A nemzetgazdasági miniszter 8/2025. (III. 31.) NGM rendelete az e-pénztárgépek forgalmazásáról, üzemeltetéséről, valamint az e-pénztárgépek és az e-nyugta kiállításának követelményeiről), illetve az aktualizált Fejlesztői dokumentáció tartalmazza a válaszokat a feltett kérdésekre.
